xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -655,9 +655,26 @@ AVDictionary *CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput()
         av_dict_set(&options, "user-agent", value.c_str(), 0);
         hasUserAgent = true;
       }
+      // other standard headers (see https://en.wikipedia.org/wiki/List_of_HTTP_header_fields)
+      else if (name == "accept" || name == "accept-language" || name == "accept-datetime" || 
+        name == "authorization" || name == "cache-control" || name == "connection" || name == "cookie" || name == "content-md5" || 
```

Comment:
cookie is already handled: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp#L671